.github/workflows/full-checks.yml

```diff
@@ -0,0 +1,62 @@
+name: "Full checks"
+
+on:
+  pull_request:
+
+jobs:
+  full-checks:
+    name: "Full CI checks"
+
+    runs-on: ${{ matrix.operating-system }}
+
+    strategy:
+      matrix:
+        dependencies:
+          - "locked"
+        php-version:
+          - "7.4"
+        operating-system:
+          - "ubuntu-latest"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2"
+
+      - name: "Install PHP"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          coverage: Xdebug
+          php-version: "${{ matrix.php-version }}"
+          ini-values: memory_limit=-1
```

Comment:
This isn't necessary, `-1` is the default; see also https://github.com/shivammathur/setup-php/releases/tag/2.4.0
> Disable memory_limit on all configurations